options/posix/generic/fcntl.cpp

```diff
@@ -69,8 +69,8 @@ int posix_fallocate(int fd, off_t offset, off_t size) {
 
 // This is a linux extension
 int name_to_handle_at(int, const char *, struct file_handle *, int *, int) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+	errno = ENOSYS;
```

Comment:
Same here, we should make this a sysdep. Also, there's an open PR that implements this for Linux, so we could drop this commit and piggy back off of that PR.

Reply:
+1, Dennis' PR already turns this into a sysdep.